pkg/cmd/pulumi/destroy.go

```diff
@@ -264,9 +264,7 @@ func newDestroyCmd() *cobra.Command {
 					" associated with this stack.\n", protectedCount)
 			} else if res == nil && len(*targets) == 0 {
 				if !jsonDisplay && !remove {
-					fmt.Printf("The resources in the stack have been deleted, but the history and configuration "+
-						"associated with the stack are still maintained. \nIf you want to remove the stack "+
-						"completely, run `pulumi stack rm %s`.\n", s.Ref())
+					fmt.Printf("Run `pulumi stack rm %s` to delete the stack history and configuration\n", s.Ref())
```

Comment:
I think the prior message clearly explains the benefits and drawbacks of the command as it may not be obvious to users who won't think much about it.

Thoughts?

Reply:
I think the reason I reacted to the change is because the first one seems that it's more mentioning the command whereas this one is sounds like it is suggesting the command.

Reply:
yah, dont feel strongly, happy to close but id be surprised if we have folks reading that entire message

Reply:
Thanks for suggesting though.

Looking at the two, I think message length is a nice usability tool. If we want to encourage the user to read the command, the message should be shorter. If we think the user might want to know about the command, make the message longer.